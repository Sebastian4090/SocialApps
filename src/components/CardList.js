import React from 'react';
import Card from './Card';



const CardList = ({ socials }) =>{
    console.log('cardlist!')

    return (
        <div>
            {socials.map((site, i) => {
            return (
            <Card 
            key={socials[i].id} 
            name={socials[i].name} 
            domain={socials[i].domain} 
            link={socials[i].link}
            />
            );
        })
        }      
        </div>
    );
}

export default CardList;