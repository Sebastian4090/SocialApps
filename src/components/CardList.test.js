import { shallow } from 'enzyme';
import React from 'react';
import CardList from './CardList';

it('expect to render CardList component', () => {
    const mockSocials = [
        {
            id : 1,
            name : 'test',
            domain: 'testSite',
            link: 'test.pl'

        }
    ]

    expect(shallow(<CardList socials={mockSocials}/>)).toMatchSnapshot()
})