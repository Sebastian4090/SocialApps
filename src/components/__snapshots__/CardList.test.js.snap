// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CardList component 1`] = `
<div>
  <Card
    domain="testSite"
    key="1"
    link="test.pl"
    name="test"
  />
</div>
`;
